test/e2e/argo-workflows/cluster-agent.yaml

```diff
@@ -416,6 +414,12 @@ spec:
                 - /opt/datadog-agent/bin/agent/agent
                 - start
                 env:
+                - name: DD_CLUSTER_AGENT
```

Comment:
Can we avoid to exclusively configure the agent by a envvar and have a proper configmap binded in?
This help us to track any issues with entrypoint and configuration files.

Reply:
I will make a card to discuss another way to configure the agent to use the DCA service mapper other than `DD_CLUSTER_AGENT`, but for now this is a good e2e test for the current state of the world.

Reply:
Do you mean for this specific e2e test to set `cluster_agent: true` in the `datadog.yaml` instead of using the env var?

Reply:
Yes exactly